controllers/datadogagent/feature/apm/const.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package apm
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	apmRBACPrefix = "apm"
+)
+
+// getRBACResourceName return the RBAC resources name
+func getRBACResourceName(owner metav1.Object) string {
+	return fmt.Sprintf("%s-%s-%s", owner.GetName(), apmRBACPrefix, "cluster-agent")
+}
```

Comment:
Not sure if it is advised to create a new clusterrole specific to APM, or if we should reuse the existing shared cluster agent clusterrole.

Reply:
I believe we use both methods so either is fine by me

Reply:
I believe we use both methods so either is fine by me